dev/run-python-skinny-tests.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+set -x
+# Set err=1 if any commands exit with non-zero status as described in
+# https://stackoverflow.com/a/42219754
+err=0
+trap 'err=1' ERR
+export MLFLOW_HOME=$(pwd)
+export MLFLOW_SKINNY='true'
+
+pytest --verbose tests/test_skinny.py
+pytest --verbose tests/tracking/service/
```

Comment:
Guessing this is just a start, but it's worth noting that we'll also want test coverage for project execution & model registry interactions.

Reply:
Yeah this is to unblock tests.

I think the target is plugin + RestStore tests.

For the above comment about nested packages, I think that is out of scope for this initial set of proposed changes

Reply:
If you have a solution of isolated server tests so we can split the envs: skinny env sdk + server env for e2e tests that would help for the next set of contributions around RestStore and plugins

Reply:
I am familiarizing myself with the test infra around the server but my current options are quite involved.

I am hoping for a simpler solution